pages/retrospective-assessment/index.js

```diff
@@ -0,0 +1,104 @@
+const { Router } = require('express');
+const bodyParser = require('body-parser');
+const { get } = require('lodash');
+const routes = require('./routes');
+const { getComments, getChangedValues } = require('../project-version/middleware');
+const extractComments = require('../project-version/lib/extract-comments');
+
+module.exports = () => {
+  const app = Router({ mergeParams: true });
+
+  app.use(bodyParser.json({ limit: '5mb' }));
```

Comment:
Not sure if this is copied from somewhere, but can we avoid the hard-coded limit and use the limit from config?

See https://github.com/UKHomeOffice/asl-pages/blob/master/pages/project-version/update/router.js#L76